Resources/public/js/models/ez-locationmodel.js

```diff
@@ -381,6 +410,17 @@ YUI.add('ez-locationmodel', function (Y) {
                     return contentInfo;
                 }
             },
+
+            /**
+             * The location's path. By default it is empty. You need to call the loadPath method to set this attribute.
+             *
+             * @attribute path
+             * @default false
+             * @type array
+             */
+            path: {
+                value: false
```

Comment:
I would set it as readOnly, since one is not supposed to change anything on it directly.